package.json

```diff
@@ -36,8 +40,10 @@
     "test:ts-js": "npx tsc --noEmit --project ./tsconfig.js.json",
     "test:ts-compatibility": "npx typescript@~3.8.2 --noEmit --project ./tsconfig.dist.json",
     "test:typings": "dtslint --localTs node_modules/typescript/lib --expectOnly typing_tests",
-    "build:ts": "tsc && gulp inject-version",
-    "build": "npm run lint && npm run clean && npm run build:ts && gulp build && npm run docs",
+    "build-es5": "rimraf es5 && tsc --target es5 --outDir es5 && gulp inject-version --buildDir es5 && gulp pack-browser --buildDir es5",
```

Comment:
I'm not sure whether there are ppl using `unpkg` or similar to get the browserify build.
I do vaguely remember that we once pointed someone on support to use that.
Not a strong opinion on this, but if we care we should use the old build directory for the `gulp pack-browser` output.

Reply:
I think if they're using a floating `unpkg` path then they've got bigger problems than the path changing, but I've updated the mentions to it